pkgs/by-name/it/itools/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, pkgs
+, pkg-config
+, perl
+, libitl
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "itools";
+  version = "1.1";
+
+  nativeBuildInputs = [ pkg-config autoconf automake ];
+  buildInputs = [ libitl perl ];
+
+  outputs = [ "out" "man" ];
+
+  src = fetchFromGitHub {
+    owner = "arabeyes-org";
+    repo = "itools";
+    rev = finalAttrs.version;
+    hash = "sha256-DxTZaq2SlEmy9k7iAdjctpPkk+2rIaF+xEcfXj/ERWw=";
+  };
+
+  preConfigure = ''
+    autoreconf -f -i
```

Comment:
Have you tried `autoreconfHook` (in `nativeBuildInputs`)?

Reply:
Was not aware that it exists. Will try and report back.